src/Stories/our-journey-of-transforming-to-a-devops-culture.md

```diff
@@ -10,7 +10,12 @@
 
 #Introduction
 ##Value for the reader to read this ebook
-// willys
+For many years we've been evaluating and implementing Agile practices across our geographically distributed, part-time, and volunteered based teams. While we shared our learnings in the [Managing Agile OSS Projects with Microsoft VSO](https://blogs.msdn.microsoft.com/microsoft_press/2015/04/09/free-ebook-managing-agile-open-source-software-projects-with-microsoft-visual-studio-online/) eBook, we realized we're not done. We need to scale and more importantly, deliver value to our developer community quickly. 
```

Comment:
"...years, we've..."
"...volunteer-based..."
"...and, more importantly, deliver..."